tests/gcp/installer/util/test_resource/target_forseti_conf_server.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
This file should not be called the `target`, but rather the `source` as source has the meaning of `from`.  If this is the case, please fix everywhere.

Also, you should call this with the 1.0 naming of `forseti_conf.yaml`.

Reply:
Updated.